renovate.json

```diff
@@ -4,7 +4,6 @@
   ],
    "ignoreDeps": [
                   "de.fraunhofer.aisec.mark:de.fraunhofer.aisec.mark",
-                  "de.fraunhofer.aisec:cpg", 
```

Comment:
I guess we should PR new releases of the CPG immediately. It will at least tell us, if it will work and act as a reminder.